MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/GiniMerchant.swift

```diff
@@ -78,8 +78,8 @@ public struct DataForReview {
      - secret: The client secret provided by Gini alongside the client ID. This is used to authenticate your application to the Gini API.
      - domain: The domain associated with your client credentials. This is used to scope the client credentials to a specific domain.
      */
-    public init(id: String, secret: String, domain: String, logLevel: LogLevel = .none) {
-        let client = Client(id: id, secret: secret, domain: domain)
+    public init(id: String, secret: String, domain: String, apiVersion: Int = 4, logLevel: LogLevel = .none) {
```

Comment:
Suggestion: Same here regarding the constant. Maybe create an enum based on SDK and provide a apiVersion Constant for each.